src/components/blocks/software/Code/ApiKeyIndicator.tsx

```diff
@@ -1,12 +1,29 @@
-import React from 'react';
+import React, { useState } from 'react';
+import Icon from '@ably/ui/core/Icon';
 
-const tooltip =
-  'This code example uses a temporary key that is rate limited and expires in 4 hours. Sign in to Ably to use your API keys instead.';
-
-const APIKeyIndicator = () => (
-  <div className="docs-api-key-label" title={tooltip}>
-    Api Key: Demo Only
-  </div>
-);
+const APIKeyIndicator = ({ tooltip }: { tooltip: string }) => {
```

Comment:
We should pull this type out into it's own object or interface, not sure what the convention is for the project.